bom/application/pom.xml

```diff
@@ -53,7 +53,7 @@
         <smallrye-metrics.version>4.0.0</smallrye-metrics.version>
         <smallrye-open-api.version>3.13.0</smallrye-open-api.version>
         <smallrye-graphql.version>2.11.0</smallrye-graphql.version>
-        <smallrye-fault-tolerance.version>6.5.0</smallrye-fault-tolerance.version>
```

Comment:
Given this is a candidate for backporting to 3.15, here is a dependency version upgrade

Reply:
Oh, I accidentlly pushed this.